### PR TITLE
Nerfs the Staff

### PR DESCRIPTION
**Pull request by ParadoxSpace**
Takes the staff of change and wand of change out of the wizard's normal gear. Adjusts the max shots and recharge time of the other staffs.

### CODE REVIEW

**Comment by ParadoxSpace** [comment]
!review

File: code/modules/projectiles/guns/energy/magic.dm [hunk 1]
```diff
@@ -70,7 +70,8 @@ obj/item/weapon/gun/energy/staff/special_check(var/mob/living/user)
 	desc = "An artefact that spits bolts of life-force which causes objects which are hit by it to animate and come to life! This magic doesn't affect machines."
```
Comment (Sindorman) [inline code/modules/projectiles/guns/energy/magic.dm]:
```suggestion
	desc = "An artifact that spits bolts of life-force which causes objects which are hit by it to animate and come to life! This magic doesn't affect machines."
```

File: code/modules/projectiles/guns/energy/magic.dm [hunk 2]
```diff
@@ -239,33 +242,6 @@ obj/item/weapon/gun/energy/staff/focus/attack_self(mob/living/user as mob)
 		return 0
 	return 1
 
-
-/obj/item/weapon/gun/energy/wand/polymorph
```
Comment (Sindorman) [inline code/modules/projectiles/guns/energy/magic.dm]: Why did you remove it?

File: code/modules/projectiles/guns/energy/magic.dm (hunk 1 above)
Comment (VTCobaltblood) [reply code/modules/projectiles/guns/energy/magic.dm]: not incorrect

File: code/modules/projectiles/guns/energy/magic.dm (hunk 2 above)
Comment (ParadoxSpace) [reply code/modules/projectiles/guns/energy/magic.dm]: Because it's part of the PR.

**Comment by MarinaGryphon** [comment]
I understand the need to replace it with a better version, but jumping straight to Thanos-snapping it is a Fowl-tier move tbh. Feedback thread required.

**Comment by ParadoxSpace** [comment]
https://forums.aurorastation.org/topic/12596-removing-the-staff-of-change/

**Comment by ToasterStrudes** [comment]
I thought the op admins and mods were finally getting what was coming to them, false advertising

File: code/modules/projectiles/guns/energy/magic.dm (hunk 2 above)
Comment (Sindorman) [reply code/modules/projectiles/guns/energy/magic.dm]: You said you were keeping it for asminbuse.

Comment (ParadoxSpace) [reply code/modules/projectiles/guns/energy/magic.dm]: No I did not.

**Comment by Bygonehero** [comment]
seems like the forum thread is ignored?

**Comment by Bygonehero** [comment]
No point in removing it. Just make it unusable offensively. Give it a massive shot delay and make its transformation gradual and not instant.

**Comment by Arrow768** [comment]
Closing in favor of #6899
